src/viz/viewer.cpp

```diff
@@ -46,7 +46,7 @@ struct Viewer::Impl {
 
     inline void loop(
         void (*input_fn)(void *, CountT, CountT, const UserInput &),
-        void *input_data, void (*step_fn)(void *), void *step_data);
+        void *input_data, void (*step_fn)(void *), void *step_data, void (*ui_fn)(void*), void* ui_data);
```

Comment:
Same as above, hard for me to tell on github but if this is over 80 chars wrap it.